builtin/providers/aws/resource_aws_iot_policy_attachment_test.go

```diff
@@ -0,0 +1,67 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSIoTPolicyAttachment_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSIoTPolicyAttachmentDestroy_basic,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccAWSIoTPolicyAttachment_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSIoTPolicyAttachmentExists_basic("aws_iot_policy_attachment.cert_policies"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckAWSIoTPolicyAttachmentDestroy_basic(s *terraform.State) error {
+	return nil
```

Comment:
The problem here is that I'm not sure how to test if the attachment was destroyed and just copied this idea from the `resource_aws_iam_policy_attachment_test.go` which does the same:

```go
func testAccCheckAWSPolicyAttachmentDestroy(s *terraform.State) error {
	return nil
}
```